$NetBSD: patch-ab,v 1.1 2010/06/27 08:51:11 obache Exp $

Gtk-WARNING **: GtkSpinButton: setting an adjustment with non-zero page size is deprecated

--- src/tomoe-reading-search.c.orig	2007-04-25 07:49:31.000000000 +0000
+++ src/tomoe-reading-search.c
@@ -203,7 +203,7 @@ tomoe_reading_search_init (TomoeReadingS
     gtk_box_pack_start (GTK_BOX (hbox), label, FALSE, FALSE, 4);
     /*gtk_widget_show (label);*/
 
-    adj = gtk_adjustment_new (1.0, 1.0, 200.0, 1.0, 5.0, 5.0);
+    adj = gtk_adjustment_new (1.0, 1.0, 200.0, 1.0, 5.0, 0.0);
     spin = gtk_spin_button_new (GTK_ADJUSTMENT (adj), 0.0, 0);
     priv->min_strokes_spin = spin;
     g_signal_connect (G_OBJECT (spin), "value-changed",
@@ -217,7 +217,7 @@ tomoe_reading_search_init (TomoeReadingS
     gtk_box_pack_start (GTK_BOX (hbox), label, FALSE, FALSE, 4);
     /*gtk_widget_show (label);*/
 
-    adj = gtk_adjustment_new (20.0, 1.0, 200.0, 1.0, 5.0, 5.0);
+    adj = gtk_adjustment_new (20.0, 1.0, 200.0, 1.0, 5.0, 0.0);
     spin = gtk_spin_button_new (GTK_ADJUSTMENT (adj), 0.0, 0);
     priv->max_strokes_spin = spin;
     g_signal_connect (G_OBJECT (spin), "value-changed",
